test/.eslintrc.json

```diff
@@ -1,13 +1,12 @@
 {
+	"root": true,
+
+	"extends": "../.eslintrc-browser.json",
+
 	"env": {
 		"browser": true
```

Comment:
I'm not sure how to best play it out. The issue is that we don't want `browser: true` to be specified in source as we take all the browser stuff from the `window` variable and that's important for jsdom compatibility. On the other hand, it would be too much to fix in tests and that would only help us if we ever tried to run the test suite (or its parts) in jsdom which we don't have plans currently. So setup between `src` & `test` will differ.

What would you suggest?